src/vs/workbench/parts/output/common/output.ts

```diff
@@ -111,6 +111,11 @@ export interface IOutputChannel {
 	output: string;
 
 	/**
+	 * Returns the value indicating whether the channel has scroll locked.
+	 */
+	hasScrollLock: boolean;
```

Comment:
I would rename this to `scrollLock`, as `outputChannel.hasScrollLock = false` sounds strange to me